dask/tests/test_tokenize.py

```diff
@@ -71,8 +71,10 @@ def check_tokenize(*args, **kwargs):
         args3, kwargs3 = cloudpickle.loads(cloudpickle.dumps((args3, kwargs3)))
 
         tok2 = tokenize(*args2, **kwargs2)
+        assert tok2 == before, (args, kwargs)
```

Comment:
this is the important test change that ensures that going through cloudpickle doesn't change the token